src/main/java/ru/yandex/practicum/filmorate/services/InMemoryHistoryUserService.java

```diff
@@ -0,0 +1,44 @@
+package ru.yandex.practicum.filmorate.services;
+
+import ru.yandex.practicum.filmorate.exception.ValidationException;
+import ru.yandex.practicum.filmorate.model.User;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class InMemoryHistoryUserService implements UserService {
+    private final HashMap<Integer, User> users = new HashMap<>();
+    int id = 0;
+
+    @Override
+    public User addUser(User user) {
+        if ((user.getName() == null) || (user.getName().equals(""))) {

```

Comment:
_Саморазвитие_
```suggestion
        if ((user.getName() == null) || (user.getName().isBlank())) {
```
`.isBlank()` - метод строки, который проверят, что строка не пустая
Вот [здесь](https://www.baeldung.com/java-blank-empty-strings) чуть больше проверки строк

Reply:
поправила